apps/prairielearn/src/tests/instructorCourseAdminSettings.test.ts

```diff
@@ -177,7 +176,7 @@ describe('Editing course settings', () => {
       },
     });
     assert.equal(response.status, 200);
-    assert.match(response.url, /\/pl\/course\/1\/edit_error\/\d+$/);
+    assert.match(response.url, /\/pl\/course\/1\/course_admin\/settings$/);
```

Comment:
It turns out that the only thing we had to do to make this work was to add a check of `editor.shouldEdit()`!

Reply:
Nice!